src/rest-server/src/controllers/v2/group.js

```diff
@@ -168,4 +191,5 @@ module.exports = {
   updateGroupDescription,
   updateGroupExternalName,
   deleteGroup,
+  updateGroupExtensionAttr,
```

Comment:
So this function can only update one key field in extension, right?

Reply:
Yes, more specificly, only one key in a nested extension